src/lib/components/ContentfulRichText/__tests__/documents/.generate-documents-from-markdown.cjs

```diff
@@ -0,0 +1,82 @@
+// Run this file to generate JSON rich text files from every markdown file in this directory
```

Comment:
Just curious, is this something we should be running as part of test setup instead of checking the `.md.html` and `.md.json` files into source?

Reply:
good point, I've made this part of the `pretest` script.